lib/diagnostic.rb

```diff
@@ -28,15 +28,27 @@
 ##
 # your response here
 ##
+class Person
+  attr_accessor :name
+  attr_reader :age
+  attr_writer :location
+def initialize
+  @name = name
+  @age = age
+  @location = locatin
+end
+end
 
 ##
 # Instatiate a new Person object using arguments "Dave", 32, and "Ohio".
 # Then, in the next line, change this Person's location property to
 # "Somerville". Finally, assign the modified Person to `Response.dave` below.
 
 ## replace nil with your response, then continue your work on the next line
-dave = nil
-Response.dave = dave
+dave = Person.new ('Dave', '32', 'Ohio')
+
+
+Response.dave = { 'Dave => Ohio'}
```

Comment:
This will cause a syntax error since it's being interpreted as a hash literal with the key `'Dave => Ohio'` with no `:` or `=>` to indicate the value.  We want to modify `dave`'s `location` and set it to `'Somerville'`.

```ruby
dave = Person.new('Dave', 32, 'Ohio')
dave.location = 'Somerville'
Response.dave = dave
```